examples/40_advanced/example_text_preprocessing.py

```diff
@@ -80,13 +80,13 @@
 # View the models found by auto-sklearn
 # =====================================
 
-print(automl.leaderboard())
+# print(automl.leaderboard())
```

Comment:
Hey, why do you add a comment here?

Reply:
the print function somehow yields an error in the auto check on git so i commented it out but i can include it again, but then it probably will raise an error

Reply:
These errors came up before and I'm not certain on the reason that it happens.

* https://github.com/automl/auto-sklearn/blob/master/autosklearn/estimators.py#L839

Essentially there is an `id` in the ensemble that is not included in the run history of SMAC. As it's model id 1, my best guess is that it is to do with the dummy model not being run by SMAC in conjucntion with some other outcome state.

My best guess is that no model found was better than the Dummy model.